smokes/e2e/force.scenarios.coffee

```diff
@@ -1,16 +1,30 @@
+# coffee script 
+# test goal: checks the capability to define a reason and to cancel/start the build
+
+forceSchedulerPage = require('./utils.coffee')
+
+describe('', () ->
+    beforeEach(() ->
+        reasonForce =  new forceSchedulerPage()
+        reasonForce.get()
+    )
+)
+
+
 describe 'force', () ->
     it 'should create a build', () ->
-        browser.get('#/builders/1')
+        reasonForce =  new forceSchedulerPage()
+        
         lastbuild = element.all(By.css('span.badge-status.results_SUCCESS')).count()
-        browser.get('#/builders/1/force/force')
-        browser.waitForAngular()
-        element(By.buttonText('Start Build')).click()
-        browser.get('#/builders/1')
+        reasonForce.getLink('1/')
+        reasonForce.getLink('1/force/force')
+        reasonForce.getStart()
```

Comment:
getStart is not clear to me.

clickStartButton() would be more meaningful.

actually, what would be better is to say

forcePage.getStartButton().click()